bang/utils.py

```diff
@@ -125,6 +125,30 @@ def generateDifficulty(difficulty):
         small_images=(u'<img src="{}" class="song-small-note">'.format(note_image) * (difficulty % 5)),
     )
 
+def bandField(band, i):
+    return {
+        'image': staticImageURL('mini_band_icon/{}.png'.format(band)),
+        'verbose_name': _('Band'),
+        'type': 'image_link',
+        'link': u'/members/?i_band={}'.format(i),
+        'ajax_link': u'/ajax/members/?i_band={}&ajax_modal_only'.format(i),
+        'link_text': band,
+        'value': staticImageURL('band/{}.png'.format(band)),
+    }  
+
+# For Event, Gacha, Song
+def subtitledImageLink(item, tl, icon, sub=None):
```

Comment:
Make sure you don't use any acronym, abbreviation or short names. I didn't know what `tl` meant here and had to read the code, it's better to be able to tell just from the variable name.